packages/high-roller-bot/src/index.ts

```diff
@@ -0,0 +1,492 @@
+import {
+  FirebaseServiceFactory,
+  MNEMONIC_PATH,
+  Node
+} from "@counterfactual/node";
+import { Node as NodeTypes } from "@counterfactual/types";
+import { ethers } from "ethers";
+import HashZero from "ethers/constants";
+import fs from "fs";
+import fetch from "node-fetch";
+import path from "path";
+import { v4 as generateUUID } from "uuid";
+
+const provider = ethers.getDefaultProvider("ropsten");
+const API_TIMEOUT = 30000;
+let bot: UserSession;
+
+let BASE_URL = `https://server.playground-staging.counterfactual.com`;
+
+console.log("Creating serviceFactory");
+let serviceFactory: FirebaseServiceFactory;
+if (process.env.TIER && process.env.TIER === "development") {
```

Comment:
Use `dotenv-extended` or `dotenv-safe`. Also even without it `process.env.TIER` should be `undefined`